### DISCUSSION TITLE
Get exact plot points

### DISCUSSION THREAD

**Discussion by MihirSeth** [theOehrly/Fast-F1#213] [Q&A]
Is there any way to get the exact plot points?

For example, for this I want to get all the exact points (x, y)
<img width="1440" alt="Screenshot 2022-07-20 at 5 16 26 PM" src="https://user-images.githubusercontent.com/52575613/179974362-2c6ea625-bca7-4a65-b8c1-02567c09c99d.png">

**Comment by MaxZec** [comment]
Hi, can I ask you how have been able to get distance on the X axle? Also yuor data seems to match perfectly while everytime I try to compare two laps there's a lot of differences in distance...

**Comment by theOehrly** [comment]
Lap and telemetry data are subclasses of pandas dataframes. So you can use all the methods that are provided by pandas to interact with the data, including, for example, export to csv and similar.

**Reply by MaxZec** [reply]
Sorry totally miss your answer.. Thnx for the reply, so with panda is possible to set the distance and finally match the data.. That's cool, I definitely need to lerain how to use it.
